sample/src/main/res/layout/activity_expansion_panel.xml

```diff
@@ -48,6 +49,12 @@
 
         </com.natura.android.expansionPanel.ExpansionPanel>
 
+        <Button
+            android:id="@+id/current_state_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Check ExpansionPanel's\ncurrent state"/>
```

Comment:
Se a gnt add um espacinho aqui o Typo para de reclamar ;)

Reply:
@mlcsouza acho que nesse caso @LucasZeta pode só ignorar essa mensagem, pois é um false-positive do Typo CI. Recomendo não adicionarmos `ncurrent` à safelist, para que o Typo CI continue detectando quando for realmente um typo. Faz sentido?